### PR TITLE
fix: id param in fetch method should be number|string

### PR DESCRIPTION
**Pull request by galuszkak**

### CODE REVIEW

**Comment by galuszkak** [comment]
This is when someone for example uses UUID rather than Integer. I think docs, are mentioning that just code wasn't align with it.

File: src/core/item.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export abstract class Item {
 	 * @param {Number|String} id
 	 * @param {FetchOptions} fetchOptions - Setting {cache: true} will ensure that the item will be fetched from cache if it exists and the HTTP request is cached.
 	 */
-	static fetch<T extends Item>(id: number, fetchOptions?: FetchOptions): Promise<T> {
+	static fetch<T extends Item>(id: number|string, fetchOptions?: FetchOptions): Promise<T> {
```
Comment (rolandjitsu) [inline src/core/item.ts]: Please add a space around `|`.

File: src/core/store.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export class Store<T extends Item> {
 
 	}
 
-	fetch(id: number, {cache = true}: FetchOptions = {}): Promise<T> {
+	fetch(id: number|string, {cache = true}: FetchOptions = {}): Promise<T> {
```
Comment (rolandjitsu) [inline src/core/store.ts]: Here we need a space around `|` as well.

**Comment by galuszkak** [comment]
@rolandjitsu fixed. Thanks for review 👍

**Comment by rolandjitsu** [comment]
The package is published on NPM now.

**Comment by galuszkak** [comment]
@rolandjitsu awesome! Thanks!

**Pull request merged**